modules/ossm-automatic-sidecar-injection.adoc

```diff
@@ -6,20 +6,24 @@
 [id="ossm-automatic-sidecar-injection_{context}"]
 = Enabling automatic sidecar injection
 
-When deploying an application into the {ProductName} you must opt in to injection by specifying the `sidecar.istio.io/inject` annotation with a value of `"true"`. Opting in ensures that the sidecar injection does not interfere with other OpenShift features such as builder pods used by numerous frameworks within the OpenShift ecosystem.
+When deploying an application, you must opt-in to injection by setting the `sidecar.istio.io/inject` annotation to `"true"`. Opting in ensures that the sidecar injection does not interfere with other OpenShift features such as builder pods used by numerous frameworks within the OpenShift ecosystem. Automatic injection using annotation removes the need to label projects.
 
 .Prerequisites
 
 * Identify the deployments for which you want to enable automatic sidecar injection.
-* Locate the application's YAML configuration file.
 
 .Procedure
 
-. Open the application's configuration YAML file in an editor.
+. Open the application's deployment configuration YAML file in an editor. To find a deployment use the `oc get` command. For example, for an app called `sleep` in the `sleep` namespace, use the following command to see the resource in YAML format.
```

Comment:
Should this be (as specified below):

Open the application's deployment configuration YAML in an editor.

It's not really a file in Kubernetes, but rather a resource. Also, would it simplify things to do:

```
oc edit deployment sleep -o yaml 
```

This would open the resource in the system's text editor without having to save to a file and apply after. 

Regardless,

```
oc get deployment sleep - yaml
```

is missing the -o. It should be:

```
oc get deployment sleep - yaml
```